packages/example-getting-started/docs/2-scaffold-app.md

```diff
@@ -0,0 +1,13 @@
+### Create your app scaffolding
+
+Install the `@loopback/cli` package. This will give you the command-line
```

Comment:
This has been covered in the prerequisite page. This line should be phrased so that it 'remind' the users to have it installed instead